### ISSUE TITLE
Case study suggestions for making Retiarii accessible

### ISSUE THREAD

**Issue by sayakpaul** [microsoft/nni#3873]
**What would you like to be added**:

More case studies on the byproducts of Neural Architecture Search:

* MobileNetV3
* MobileDet
* Swish

These case studies do not need to fully implemented. Minimal demonstrations should be sufficient enough for users to expand the case studies. 

Also being able to specify hardware-aware constraints in the search space is an important use-case for NAS. An example showing how to do that would be useful. 

**Why is this needed**:

Without establishing NNI's Retiarii capabilities transfer well to the existing line of works, it might be hard for researchers to adapt Retiarii for their own use.

**Comment by QuanluZhang** [comment]
@sayakpaul , thanks for your great suggestions. we have an WiP pr [here](https://github.com/microsoft/nni/pull/3868) which implemented the space of Swish. We expect users to directly use this module in their PyTorch/TensorFlow model to automatically find out the best activation function specifically for their model.

**Comment by sayakpaul** [comment]
Great. I will look into it for sure. Should I keep this issue open for other suggestions?

**Comment by QuanluZhang** [comment]
> Great. I will look into it for sure. Should I keep this issue open for other suggestions?

yes, we can keep it open for now :)

**Comment by sayakpaul** [comment]
Sure, thank you. You might want to add a relevant label to this issue for better/easier triage.

**Comment by ultmaster** [comment]
Added your vote for MobileNetV3 and MobileDet. Will consider them in future planning. :)

**Comment by sayakpaul** [comment]
Sure. I personally think teaching the developers how to go about constructing informative search spaces itself is very rewarding.
